### PR TITLE
fix Issue 14697 add PCLMULQDQ instruction to inline assembler

### PR DESCRIPTION
**Pull request by WalterBright**
Interestingly it was in the tables but never referred to.

### CODE REVIEW

**Comment by MoonlightSentinel** [comment]
Tests?

**Comment by WalterBright** [comment]
See the test added to test/runnable/iasm64.d

**Comment by MoonlightSentinel** [comment]
Sorry, commented on the wrong PR.

**Review by ghost — APPROVED** [review]

**Review by thewilsonator — APPROVED** [review]

**Comment by NilsLankila** [comment]
wait for the stable-priority PRs that are in auto merge state before merge this one PLZ.

**Comment by NilsLankila** [comment]
well NVM, let's merge. Prioritizing

1. does not work
2. Martin has already put the tag for 2.093.1

**Pull request merged**
